src/tests/Equinor.ProCoSys.IPO.Command.Tests/InvitationCommands/CreateInvitation/CreateInvitationCommandHandlerTests.cs

```diff
@@ -116,30 +175,53 @@ public async Task Invitation_is_added_to_repository_test()
                 _mcPkgScope,
                 null);
 
-            await dut.Handle(command, default);
+            _dut = new CreateInvitationCommandHandler(
+                _plantProviderMock.Object,
+                _meetingClientMock.Object,
+                _invitationRepositoryMock.Object,
+                _unitOfWorkMock.Object,
+                _commPkgApiServiceMock.Object,
+                _mcPkgApiServiceMock.Object,
+                _personApiServiceMock.Object,
+                _functionalRoleApiServiceMock.Object);
+        }
+
+        [TestMethod]
+        public async Task Invitation_is_added_to_repository_test()
+        {
+            await _dut.Handle(_command, default);
 
             Assert.IsNotNull(_createdInvitation);
             Assert.AreEqual(1, _saveChangesCount);
         }
 
         [TestMethod]
-        public async Task Meeting_is_created_and_meeting_id_is_set_on_invitation_test()
+        public async Task McPkgs_are_added_to_invitation()
```

Comment:
I know this wasn't your doing but I think method names should follow the style of our other test classes. 
HandleCreateInvitationCommand_ShouldAddMcPkgsToInvitation(), etc.